templates/profile.html

```diff
@@ -11,13 +11,49 @@ <h1>Create a profile</h1>
   <section>
     <h2>About</h2>
     <img src=https://rawgit.com/cafebabel/styleguide/master/img/profile-pic-placeholder.jpg alt="Profile pic placeholder"> <!-- #US14 https://github.com/cafebabel/cafebabel.com/issues/14 -->
-      <ul>
-          <li><a data-edit-name=socials-twitter href="{{ user.profile.socials.twitter }}" class=icon-twitter></a></li>
-          <li><a data-edit-name=socials-facebook href="{{ user.profile.socials.facebook }}" class=icon-facebook></a></li>
-          <li><a data-edit-name=socials-instagram href="{{ user.profile.socials.instagram }}" class=icon-instagram></a></li>
-          <li><a data-edit-name=socials-medium href="{{ user.profile.socials.medium }}" class=icon-medium></a></li>
-          <li><a data-edit-name=socials-flickr href="{{ user.profile.socials.flickr }}" class=icon-flickr2></a></li>
-          <li><a data-edit-name=socials-pinterest href="{{ user.profile.socials.pinterest }}" class=icon-pinterest2></a></li>
+      <ul class=social-networks>
+          <li>
+              <a data-edit-name=socials-twitter href="{{ user.profile.socials.twitter }}" class=icon-twitter></a>
+              <label for=social-twitter>
+                  <span class=icon-twitter></span>
```

Comment:
Why do we have `<a class=icon-twitter>` followed by `<span class=icon-twitter></span>`?

Reply:
Cause I want twice the icon. I'm sure it's not the best way to do it, but I had no other idea yet

Reply:
There should be other ways indeed but if you want it twice it should do for now